pkgs/applications/misc/gnss-sdr/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv, fetchFromGitHub
+, armadillo
+, blas
```

Comment:
I'm a little skeptical if including `blas` and `lapack` as buildInputs is correct. `armadillo` links to a different BLAS version (our `openblasCompat`). I suspect that they should be purged.